Gifski/Preview/PreviewRenderer.swift

```diff
@@ -25,12 +28,14 @@ actor PreviewRenderer {
 		previewFrame: SendableTexture,
 		outputFrame: SendableCVPixelBuffer,
 		fragmentUniforms: CompositePreviewFragmentUniforms
-	) async throws {
+	) async throws(MTLCommandQueueRenderError<PreviewRenderer.RenderPreviewError>) {
```

Comment:
My fault, I should have been clearer. I meant to only use typed throws when the method throws a single typed error. It doesn't make sense when there are multiple error and wrapping is required.

It's not a huge benefit with typed throw, but for small methods that throw a single error, the typed-ness can be used as documentation and it also lets us use the `throw .errorName` syntax.